bdk-ffi/src/esplora.rs

```diff
@@ -13,7 +13,10 @@ use bdk_wallet::chain::spk_client::SyncResult as BdkSyncResult;
 use bdk_wallet::KeychainKind;
 use bdk_wallet::Update as BdkUpdate;
 
+use bitcoin_ffi::Txid;
```

Comment:
I feel like you don't need this bitcoin_ffi::Txid type imported since you're not using the bitcoin_ffi type really. A cleaner import would be `use bdk_wallet::bitcoin::Txid;`

Reply:
great call, updated-